.circleci/config.yml

```
@@ -651,7 +668,12 @@ jobs:
       - run:
           name: 'Run minimal tests'
           command: |
-            gosu circleci make -C src/test/regress << parameters.make >> EXTRA_TESTS="$(for i in $(seq << parameters.runs >> | circleci tests split); do echo -n '<< parameters.test >> ' ; done)"
+            tests_array=($tests)
+            for test in "${tests_array[@]}"
+            do
+                echo $test
+                gosu circleci src/test/regress/citus_tests/run_test.py -p $test -r << parameters.runs >> --use-base-schedule
```

nit: for documentation purposes I like scripts to use long flags when available. Short flags are nice for interactive usage to reduce typing, but long flags communicate the purpose much better without people requiring to look them up.

```suggestion
                gosu circleci src/test/regress/citus_tests/run_test.py --path $test --runs << parameters.runs >> --use-base-schedule
```